packages/vue/package.json

```diff
@@ -12,8 +12,8 @@
   ],
   "main": "src/index.ts",
   "dependencies": {
-    "@cosmjs/proto-signing": "^0.26.5",
     "@cosmjs/encoding": "^0.27.0",
+    "@cosmjs/proto-signing": "^0.26.5",
```

Comment:
a) It's best if all `@cosmjs/*` packages are at the same version otherwise each one will pull in its specific version dependencies
b) I suggest you use exact versions for @cosmjs rather than `^` as breaking changes are common

Reply:
for sure! thanks!